tests/recorder/recorder.go

```diff
@@ -0,0 +1,233 @@
+package main
```

Comment:
`autoplay` works for me.  I am in the process of making the abstractions necessary to make it a generic replay tool.  I will update the PR when it is done.